roles/upload-site/tasks/main.yml

```diff
@@ -120,6 +120,7 @@
     # Must truncate this table otherwise users will lose roles in WMD->SSP upgrade path.
     - "sqlq 'truncate table webauth_roles_history'"
     - "sspwmd"
+    - "-y pm-uninstall webauth_extras"
```

Comment:
This module needs to be disabled before it can be uninstalled.

Reply:
Why not roll this up in to lines 117 and 118?

Reply:
See comment on line 122.

Reply:
`GOTO 122`

Reply:
Oh, I see your note in Jira re: disabled due to webauth being disabled. How about a belt and suspenders approach and we add it to lines 117 and 118 anyway?

Reply:
Gotcha, needs it for drush command.

Reply:
Well, we don't technically need `webauth_extras` for the drush command, but whatevs, it's merged now.